TOMP-API.yaml

```diff
@@ -1333,19 +1331,21 @@ components:
         - $ref: '#/components/schemas/period'
       properties:
         from:
-          $ref: '#/components/schemas/coordinates'
+          $ref: '#/components/schemas/coordinate'
         radius:
           description: Maximum distance a user wants to travel to reach asset in metres, e.g. 500 metres
           type: number
         to:
-          $ref: '#/components/schemas/coordinates'
+          $ref: '#/components/schemas/coordinate'
         travellers:
           description: the amount of people that have to travel from `from` to `to` [https://github.com/efel85/TOMP-API/issues/56]
           type: number
-        use-asset:
+        useAssets:
           description: when you use the /operator/available-assets and you want to book a displayed asset, you must be able to request o a planning-option for the specific asset (with provide-ids=true), post a booking with the provided id and send directly a commit. This field should contain the asset to book.
-          type: string
-        provide-ids:
+          type: array
+          items:
+            type: string
```

Comment:
The description should be changed to reflect the new array, and the items should have a format: asset id or type-of-asset id thing added so it is clear that it's not an open text field

Reply:
I think this should be it:
There are 2 scenarios for this field: 
when you use the /operator/available-assets and you want to book a specific displayed asset, you must be able to request a planning-option for the specific asset (with provide-ids=true), post a booking with the provided id and send directly a commit. This field should contain the asset to book (an array with one asset-id).
The other scenario is when having done a bluetooth swipe, you will have to post all found bluetooth devices (UUID). In that case, this array will contain UUIDs that the TO can use to filter the results.